src/main/java/com/sourcegraph/javagraph/DepresolveCommand.java

```diff
@@ -1,36 +1,81 @@
 package com.sourcegraph.javagraph;
 
+import com.beust.jcommander.Parameter;
 import com.google.gson.Gson;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
 import java.util.ArrayList;
 
 public class DepresolveCommand {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DepresolveCommand.class);
+
+    @Parameter(names = {"--debug-unit-file"}, description = "The path to a source unit input file, which will be read as though it came from stdin. Used to mimic stdin when you can't actually pipe to stdin (e.g., in IntelliJ run configurations).")
+    String debugUnitFile;
+
     public void Execute() {
-        // Read in source unit from stdin.
+
         SourceUnit unit = null;
         try {
-            InputStreamReader reader = new InputStreamReader(System.in);
-            unit = new Gson().fromJson(reader, SourceUnit.class);
-            reader.close();
+            Reader r;
+            if (!StringUtils.isEmpty(debugUnitFile)) {
+                if (LOGGER.isDebugEnabled()) {
```

Comment:
This conditional seems unnecessary. Calling `LOGGER.debug` should be a no-op if debug is not enabled, and there is minimal performance overhead since you're using a parameterized statement. (http://www.slf4j.org/faq.html#logging_performance).